.deploy/gunicorn.conf.py

```
@@ -23,6 +58,35 @@
 
+def on_starting(server):
+    """
+    Called once in the master process before workers are forked.
+
+    Wipe and recreate the prometheus multiproc dir so stale per-pid files
+    from a previous run don't pollute aggregated metrics.
+    """
+    if os.path.isdir(_PROM_DIR):
+        shutil.rmtree(_PROM_DIR)
+    os.makedirs(_PROM_DIR, exist_ok=True)
+    server.log.info(f"Prometheus multiproc dir initialised: {_PROM_DIR}")
```

Ruff (0.15.10)

[warning] Logging statement uses f-string

(G004)